src/batched-json.js

```diff
@@ -60,18 +60,20 @@ export function batchFetchJsonFor(
     // Throw user error if this element is performing URL substitutions
     // without the soon-to-be-required opt-in (#12498).
     if (opt_urlReplacement == UrlReplacementPolicy.OPT_IN) {
-      urlReplacements.collectUnwhitelistedVars(element).then(unwhitelisted => {
-        if (unwhitelisted.length > 0) {
```

Comment:
I noticed this throughout the codebase. the usage of [].length > 0 vs just
```
    if (unwhitelisted.length) {
    ...
    }
```
which should suffice here and below as 0 will coerce to false and a positive number to true, unless the list is expected to have a negative length at any point where the value will then coerce to true. Are we just explicit to be safe?

Reply:
`length` is generally understood in JS to be zero-or-positive (it commonly signifies array-likes for iteration algorithms, and that's what they expect).

But, I generally like doing explicit numeric comparisons. The minifier will minify it either way.